### DISCUSSION TITLE
Calendar Not Populating

### DISCUSSION THREAD

**Discussion by schellbell** [photoprism/photoprism#3364] [Q&A]
Hello! 
I started using PhotoPrism a couple of months back and have run into an issue where my calendar doesn't populate. There used to be an album in there from Jan 1980 filled with photos that didn't have the correct metadata associated, but after I went through and fixed those and re-indexed that album disappeared and no new ones have populated. I have 70k+ photos from the last 20 years on there and when I search by year/month/date they show up, but my calendar remains empty. 
I have tried doing complete rescans a couple of times to see if that would trigger it, but no matter what I do I can't get anything to show up in there. Has anyone had similar issues and if so how did you fix them?

**Comment by lastzero** [comment]
There might be an issue extracting the date information from your files. We recommend using exiftool to check what metadata is there.

**Reply by schellbell** [reply]
I thought it might have been that, too, so I randomly went through and tested various photos to see what exiftool found and it seems to be detecting all the information as I would expect it to 
![image](https://user-images.githubusercontent.com/26511627/236008079-449e19af-87ba-43bb-a7fb-d5a0c889c43d.png)
I've also checked and photoprism is seeing the same information in the actual file details
![image](https://user-images.githubusercontent.com/26511627/236008362-06b6666c-e099-472e-b301-53409e30cf15.png)

**Reply by lastzero** [reply]
What database and database version do you use?

**Reply by schellbell** [reply]
I'm currently using MariaDB 10.6.12

**Reply by lastzero** [reply]
Not sure what it could be then. Did you migrate from a different version or database? Did you disable the background worker?

**Reply by lastzero** [reply]
Feel free to send us file samples for testing, see our contact page for details.

**Reply by schellbell** [reply]
No this was all a clean install when I first set it up in late Feb. My last hope is to remove photoprism, clear the database, and then restart it from scratch but if that doesn't do it then I guess I'll just get by without the Calendar ¯\_(ツ)_/¯

**Reply by lastzero** [reply]
Do you have Moments disabled?

**Comment by schellbell** [comment]
Delayed update to this: I ended up clearing out the database, re-indexing everything, and now it seems the calendar (and moments) are working correctly.

**Reply by vween** [reply]
What do you mean by clearing the database?  Did you manually delete all the data?
I'm having the same issue with Calendar and Moments.
